app/helpers/application_helper/button/storage_service_new.rb

```diff
@@ -0,0 +1,21 @@
+class ApplicationHelper::Button::StorageServiceNew < ApplicationHelper::Button::Basic
+  def calculate_properties
+    super
+    if disabled?
+      self[:title] = _("No storage providers support creating storage services.")
+    end
+  end
+
+  # If we selected specific EMS disable create button if not supporting create action.
+  # Else disable button if no active providers support create action
+  def disabled?
+    if @view_context.params["id"]
+      current_ems = ExtManagementSystem.where(:id => @view_context.params["id"]).first
+      !current_ems.class_by_ems("StorageService")&.supports?(:create)
+    else
+      ExtManagementSystem.none? do |ems|
+        ems.class_by_ems("StorageService")&.supports?(:create)
+      end
```

Comment:
Double-check me on this but I think `StorageService.providers_supporting(:create).none?` would work better (it filters EMS by type-supporting first so more performant)